src/library/scala/reflect/Manifest.scala

```diff
@@ -26,13 +26,13 @@ import scala.collection.mutable.{ArrayBuilder, ArraySeq}
  *    def arr[T](implicit m: Manifest[T]) = new Array[T](0) // compiles
  *    def arr[T: Manifest] = new Array[T](0)                // shorthand for the preceding
  *
- *    // Methods manifest, classManifest, and optManifest are in [[scala.Predef]].
+ *    // Methods manifest and optManifest are in [[scala.Predef]].
  *    def isApproxSubType[T: Manifest, U: Manifest] = manifest[T] <:< manifest[U]
  *    isApproxSubType[List[String], List[AnyRef]] // true
  *    isApproxSubType[List[String], List[Int]]    // false
  *
- *    def methods[T: ClassManifest] = classManifest[T].erasure.getMethods
- *    def retType[T: ClassManifest](name: String) =
+ *    def methods[T: Manifest] = manifest[T].runtimeClass.getMethods
```

Comment:
Potentially could use `ClassTag` (which I think is more recommended?) instead

Reply:
hmmm 🤔 This is the `scala.reflect.Manifest` scaladoc. `scala.reflect.Manifest` is not deprecated.

https://github.com/scala/scala/blob/a0026305125dcc814097b1fbb798fe925ca9f81f/src/library/scala/reflect/Manifest.scala#L42-L44

Reply:
Yeah, the situation around Manifest and ClassTag is unclear and slightly confusing.. This will have to be cleaned up for Scala 3.

In the meantime, this PR LGTM. Thanks, @xuwei-k!